src/inputs/TextFieldBase.tsx

```diff
@@ -310,3 +331,7 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
     </>
   );
 }
+
+// Css that would be applied if using react-aria <VisuallyHidden />
+const visuallyHidden = Css.add("clip", "inset(50%)").add("clipPath", "").add("border", 0).hPx(1).mPx(-1).wPx(1).nowrap
+  .p0.overflowHidden.absolute.$;
```

Comment:
**suggestion:** This could be a good style to add to our `trussConfig` at the root of this repo. Then we could do something like `Css.visuallyHidden.$`

Reply:
added!